Tools/YamlCreate.ps1

```diff
@@ -739,7 +775,9 @@ Write-Host "Yaml file created: $InstallerManifestPath"
 
 Function Write-WinGet-LocaleManifest {
 $LocaleManifest = @(
+'# Created using YamlCreate.ps1'
```

Comment:
># Created using YamlCreate.ps1 [](http://example.com/codeflow?start=1&length=30)

Might want to make this a variable instead of hardcoding it multiple times

Reply:
We'll its not like its going to change any time soon